zinit.zsh

```diff
@@ -143,6 +143,20 @@ ZINIT_2MAP=(
     PZT:: https://raw.githubusercontent.com/sorin-ionescu/prezto/master/
     PZTM:: https://raw.githubusercontent.com/sorin-ionescu/prezto/master/modules/
 )
+
+# Global aliases for quick access to zinit dirs
+alias -g BIN_DIR=${ZINIT[BIN_DIR]} ZBIN=${ZINIT[BIN_DIR]} \
```

Comment:
ZBINDIR? At least when i great bin dir, I would assume /usr/bin or $HOME/bin or so...

Reply:
Ok, I've went for a general `ZI…` prefix, so that the aliases are now:
`ZIBIN`, `ZIDIR`, `ZIPLUGS`, `ZISNIPS`,  etc.